frontend/amundsen_application/static/js/components/ResourceListItem/styles.scss

```diff
@@ -14,6 +14,17 @@ $resource-item-height: 110px;
   padding: $spacer-3;
   text-decoration: none;
 
+  &.is-disabled {
+    pointer-events: none;
+    cursor: not-allowed;
+    .resource-icon {
+      filter: grayscale(80%);
+    }
+    .resource-info .resource-info-text .resource-name {
```

Comment:
Would this work with just '.resource-name'?

Reply:
it doesn't I tried to use a less specific one and the purple styling kept coming through